datalake/src/main/java/com/sequenceiq/datalake/service/SdxEventsService.java

```diff
@@ -76,7 +76,7 @@
 
     private List<CDPStructuredEvent> retrieveDatalakeServiceEvents(List<StructuredEventType> types, List<String> datalakeCrns, PageRequest pageable) {
         Page<CDPStructuredEvent> pagedResponse = cdpStructuredEventDBService.getPagedEventsOfResources(types, datalakeCrns, pageable);
-        if (pagedResponse != null && pagedResponse.getContent() != null && pagedResponse.getContent().size() > 0) {
+        if (pagedResponse != null && pagedResponse.getContent().size() > 0) {
```

Comment:
Why'd you remove this null check?

Reply:
The idea was complaining that getContent() will not return null.

Reply:
Ah I was unfamiliar with how spring defines a Page